src/commands/CmdAnnotate.cpp

```diff
@@ -70,6 +70,9 @@ int CmdAnnotate::execute (std::string&)
   // Accumulated project change notifications.
   std::map <std::string, std::string> projectChanges;
 
+  if(filtered.size() > 1) {
+    std::cout << "This command will alter " << format(filtered.size()) << " tasks." << std::endl;
```

Comment:
Taskwarrior uses [verbosity tokens](https://taskwarrior.org/docs/verbosity.html) to allow users to control the amount of output and feedback messages the `task` commands are providing.

Hence we should not output this directly to `cout`, but rather make this output appear under the appropriate verbosity token. Looking at the list the `affected` seems to be the right match here (from `man taskrc`):

```
                  affected   Reports 'N tasks affected' and similar
```

There is already a convenience function called `feedback_affected` that does the check for you:

```c++
      feedback_affected ("This command will alter {1} tasks.", count);
```

Reply:
Hi,
thanks - as you've seen I'm new to contributing to taskwarrior and still finding my way through the codebase. I included your request in the next commit. It is very positive for me to see that there are quick responses to pull requests - this definetely increases my motivation to contribute - thank you again for that.

Reply:
No worries! I'm happy to see new contributors to the project. Feel free to shoot me an email if you have any questions :slightly_smiling_face: